samza-test/src/main/java/org/apache/samza/test/framework/system/CollectionStreamSystemSpec.java

```diff
@@ -32,11 +33,14 @@
  * Following system level configs are set by default
```

Comment:
Might help to give small description on usage of CONFIG_OVERRIDE_PREFIX here. 

Also this CONFIG_OVERRIDE_PREFIX is used just in the context of SystemDescriptor and StreamDescriptor ? Does this apply as general config resolution (at ConfigRewriter level) ?

Reply:
Applies as a general config resolution mechanism. See "Stream Configuration Generation" section in https://cwiki.apache.org/confluence/display/SAMZA/SEP-14%3A+System+and+Stream+Descriptors for more details.